lib/widgets/message_list.dart

```diff
@@ -438,30 +438,41 @@ class ScrollToBottomButton extends StatelessWidget {
   }
 }
 
-class MarkAsReadWidget extends StatelessWidget {
+class MarkAsReadWidget extends StatefulWidget {
```

Comment:
```
msglist: Use scale animation for mark-as-read button i pressed state
```

When Vlad [said](https://chat.zulip.org/#narrow/stream/48-mobile/topic/messages.20shift.20on.20mark-as-read/near/1813212) "i pressed state", I think he probably meant "in pressed state". 😅

Reply:
That explains everything!😅